openedx/features/survey_report/context_processors.py

```diff
@@ -0,0 +1,31 @@
+"""
+This is the survey report contex_processor modules
+
+This is meant to determine the visibility of the survey report banner
+across all admin pages in case a survey report has not been generated
+
+"""
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta  # for months test
+from .models import SurveyReport
+
+
+def admin_extra_context(request):
```

Comment:
Random thought (completely optional): Is it easy to check if this request is for an admin view and return `{'show_survey_report_banner': False}` if it's not an admin view (without having to check the database)? In that case, you might be able to remove the changes that require bumping the number of queries in the other tests.

Reply:
I implemented the change requested with this change we should revert back the query counts by 1 in all the tests? Or just on particular ones?

Reply:
We will know for sure once the tests run completely but my guess is that for all of them.